java/aquality-selenium-template/src/test/java/aquality/selenium/template/glue/stepdefinitions/DemoSteps.java

```diff
@@ -0,0 +1,61 @@
+package aquality.selenium.template.glue.stepdefinitions;
+
+import aquality.selenium.browser.AqualityServices;
+import aquality.selenium.template.configuration.Configuration;
+import aquality.selenium.template.forms.TopBarMenu;
+import aquality.selenium.template.forms.pages.ContactUsPage;
+import aquality.selenium.template.forms.pages.MainPage;
+import aquality.selenium.template.models.ContactUsInfo;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static aquality.selenium.template.forms.TopBarMenu.Item.CONTACT_US;
+
+public class DemoSteps {
```

Comment:
would you like split the steps to several classes?
I think, user could take it as example and keeps all his steps in one class referring to this example

Reply:
I beleive the end user will be smart enough to split steps into different classes;)

Reply:
I just think we don't need to give them a chance to make smth incorrect